src/stories/tests-factory/create-regression-scenarios.ts

```diff
@@ -0,0 +1,75 @@
+import chunk from 'lodash/chunk';
```

Comment:
a utility for generating regressive tests

Reply:
We don't plan to use it, let's remove